skimage/draw/draw.py

```diff
@@ -693,3 +693,75 @@ def bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
     """
     return _bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape)
+
+
+def rectangle(start, end=None, extent=None, shape=None):
+    """Generate coordinates of pixels within an nd-rectangle. Either
```

Comment:
This should be a one sentence summary.  Second sentence can be split off into another paragraph.